src/tests/encrypt-no-change/in/.keboola/manifest.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Funkcionalny test mi pride prilis rozsiahly, uplne by stacil:
- jeden config bez rows
- jeden config s 2 rows

`blocks` nemusis riesit, pretoze sa interne mapuju na `config` a to je uz otestovane.
Priamo v SQL/Python kode nemozes mat ziadnu sifrovanu hodnotu, teda niet tam velmi co testovat, oproti beznemu configu (alebo je?)

Reply:
v kode transformacie nemoze byt zasifrovana hodnota. OK dam tie transformacie prec z testov.